ntbs-service/DataAccess/NotificationRepository.cs

```diff
@@ -77,7 +77,7 @@ public IQueryable<Notification> GetRecentNotificationsIQueryable()
             return GetNotificationsWithBasicInformationIQueryable()
                 .Where(n => n.NotificationStatus == NotificationStatus.Notified
                             || n.NotificationStatus == NotificationStatus.Closed)
-                .OrderByDescending(n => n.SubmissionDate);
+                .OrderByDescending(n => n.NotificationDate);
```

Comment:
It's upsetting that this change hasn't made a test fail... do you know if that's because no test exists, or because a test does exist, but the data it's using is setup such that the orderings are equivalent?

Reply:
No tests exist. There are no unit tests for the `NotificationRepository`, and no integration tests check the contents of the notification lists. I could add some unit tests maybe? I would probably want to wait for Leon's UI tests to get his testability improvements like IDs on most DOM objects before I write them

Reply:
I think there are two options - we can either add unit tests for the `NotificationRepository` or update the `IndexPageTests` to use a real repository, rather than mocking it out. I'd be tempted to go for the latter, as it's less implementation dependent... but the former gives a more lightweight test so I'd be happy either way.

I don't think you need to wait for Leon's changes - the `IndexPageTests` are testing the model rather than the page, which is probably right for this. Testing this in the UI tests would be tricky, as we can't really control which notifications are the most recent.

Reply:
I've gone with the second approach as discussed yesterday. I've added a test fixture to store the context between the tests in the class.